example_dags/example_amazon_s3_snowflake_transform.py

```diff
@@ -21,17 +21,17 @@ def combine_data(center_1: Table, center_2: Table):
 @aql.transform()
 def clean_data(input_table: Table):
     return """SELECT *
-    FROM {{input_table}} WHERE TYPE NOT LIKE 'Guinea Pig'
+    FROM {{input_table}} WHERE type NOT LIKE 'Guinea Pig'
     """
 
 
-@aql.dataframe(identifiers_as_lower=False)
+@aql.dataframe()
```

Comment:
We should be able to handle capitalization more consistently across dataframes / loading file after implementing:
https://github.com/astronomer/astro-sdk/issues/564